virtio-queue/src/desc/mod.rs

```diff
@@ -0,0 +1,61 @@
+//! Descriptor types for virtio queue.
+
+use vm_memory::{ByteValued, Le16, Le32, Le64};
+
+pub mod split;
+
+/// A virtio descriptor's layout constraints with C representation.
+/// This is a unified representation of the memory layout order
+/// for packed descriptors and split descriptors.
```

Comment:
We should document that this type corresponds to `struct virtq_desc` and add a link to the VIRTIO spec <https://docs.oasis-open.org/virtio/virtio/v1.3/csd01/virtio-v1.3-csd01.html#x1-430005>